torchci/rockset/commons/__sql/flaky_tests.sql

```diff
@@ -0,0 +1,27 @@
+SELECT
+	test_run.name,
+	test_run.classname as suite,
+	test_run.file,
+	SUM(ELEMENT_AT(JSON_PARSE(REPLACE(test_run.skipped.message, 'True', 'true')), 'num_green')) as numGreen,
+	SUM(ELEMENT_AT(JSON_PARSE(REPLACE(test_run.skipped.message, 'True', 'true')), 'num_red')) as numRed,
+	ARRAY_AGG(job.name) as jobNames,
+	ARRAY_AGG(job.id) as jobIds,
+	ARRAY_AGG(workflow.id) as workflowIds,
+	ARRAY_AGG(workflow.name) as workflowNames,
+	ARRAY_AGG(workflow.head_branch) as branches,
+    ARRAY_AGG(test_run.workflow_run_attempt) as runAttempts
+FROM
+    commons.workflow_job job 
+    INNER JOIN commons.test_run ON test_run.job_id = job.id HINT(join_strategy=lookup)
```

Comment:
For performance, which side each table is on the join matters. It's generally better to make the biggest table the left-most (e.g. put `test_run` first in the join). But if you already tried it and this is faster, that's fine.

Reply:
I'll test the two again--I did try switching up these orders before and didn't remember seeing a difference, but I will retest to make sure.